content/templates/pages/modify_req_props.md

```diff
@@ -3,3 +3,79 @@ hidden: true
 ---
 
 {{< template-page "modify_req_props" >}}
+
+# Rundown
+One gotcha that Workers developers commonly encounter is with the ergonomics of the `Request` and `Response` constructors.
+
+Unlike in the browser, it is common that a Workers developer needs to change one or more parts of an intercepted Request before passing it into a `fetch` call. It is common for developers to attempt to manually declare each part of the new Request/Response, an approach that is prone to error.
+
+**The best practice is to always use the original request to construct the new request, thereby cloning all the attributes except those you intend to change.**
+
+<div class="grey copy-group">
```

Comment:
this is cool! are this the HTML from the new code sample design? any idea if we can use a partial or something for this?

Reply:
I am actually just going to delete the HTML once we get #528